src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -24,6 +25,9 @@ public interface IInputFileResolver
     /// </summary>
     public class InputFileResolver : IInputFileResolver
     {
+        private static readonly Regex PropertyRegex = new Regex(@"\$\(([a-zA-Z_][a-zA-Z0-9_\-.]*)\)");
```

Comment:
Since this and the EmptyProperties member are only used in ReplaceMsbuildProperties I would rather have them be local to that function

Reply:
Understood I jus wanted to prevent parsing of the expression for every call of the method.

Reply:
Done, I removed the `EmptyProperties` as it was no longer needed the tests were fine after adding the `Properties` in the test cases. Buildalyzer will never put `null` there (right?) so we should be safe.